python/cucim/src/cucim/skimage/filters/ridges.py

```diff
@@ -58,7 +58,7 @@ def meijering(
     wrinkles, rivers. It can be used to calculate the fraction of the
     whole image containing such objects.
 
-    Calculates the eigenvectors of the Hessian to compute the similarity of
+    Calculates the eigenvalues of the Hessian to compute the similarity of
```

Comment:
Was this just a typo or was there some kind of behavior change?

Reply:
just a typo fix